### ISSUE TITLE
Spherical Motion Dynamics of Deep Neural Networks with Batch Normalization and Weight Decay

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#38]
https://arxiv.org/abs/2006.08419

我々は，バッチ正規化(BN)と重み減衰(WD)を用いたディープニューラルネットワーク(DNN)の学習ダイナミクスをSMD(Spherical Motion Dynamics)と名付けて包括的に明らかにした．我々の定理は、BNによる重みのスケール不変性とWDの正則化効果に基づくものである。SMDは重みの最適化軌道が球面運動のようであることを示し、BNとWDを用いたDNNの更新効率を測定するための新しい指標である角度更新を提案した。本研究では、角度更新が事前に定義されたハイパーパラメータ（学習率、WDパラメータ、運動量係数）によってのみ決定されることを厳密に証明し、それらの定量的な関係を提供する。

最も重要なことは、SMDの定量的な結果が、ImageNetやCOCOのような複雑で大規模なコンピュータビジョンタスクにおいて、標準的なトレーニングスキームを用いて経験的な観察と完全に一致することです。また、消失や爆発的な勾配の回避、シャープなミニマムに陥る危険性がないこと、学習率を縮小した場合の損失の急激な減少など、BNに関するいくつかの現象についても、全く新しい視点から合理的な解釈を得ることができる。さらに、SMDの実用的な意義を提示するために、SMDと一般的な学習率調整スキームとの関連性を議論する。線形スケーリング原理

ーーーーーーーーーーーーー
この概念で考えるとSharp Flat意識しなくてよいということ？ @mtkwT
